jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/layouts/BoxTests.java

```diff
@@ -0,0 +1,47 @@
+package io.github.epam.material.tests.layouts;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.tools.LinqUtils.safeException;
+import static io.github.com.StaticSite.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.testng.Assert.fail;
+
+public class BoxTests extends TestsInit {
+
+    @Test
+    public void containedBoxTest() {
+        defaultBoxPage.open();
+
```

Comment:
need add assert for attributes element1, element2, element3 with cheking values "Object object"